transfers/transfer.py

```diff
@@ -491,41 +541,76 @@ def main(am_user, am_api_key, ss_user, ss_api_key, ts_uuid, ts_path, depth, am_u
+    parser.add_argument(
+        '--depth', '-d', help='Depth to create the transfers from relative to \
+        the transfer source location and path. Default of 1 creates transfers \
+        from the children of transfer-path.', type=int, default=1)
+    parser.add_argument('--am-url', '-a', metavar='URL',
+                        help='Archivematica URL. Default: %s' %
+                        defaults.DEF_AM_URL,
+                        default='%s' % defaults.DEF_AM_URL)
+    parser.add_argument('--ss-url', '-s', metavar='URL',
+                        help='Storage Service URL. Default: %s' %
+                        defaults.DEF_SS_URL,
+                        default='%s' % defaults.DEF_SS_URL)
+    parser.add_argument(
+        '--transfer-type', metavar='TYPE', help="Type of transfer to start. \
+        One of: 'standard' (default), 'unzipped bag', 'zipped bag', 'dspace'.",
+        default='standard', choices=['standard', 'unzipped bag', 'zipped bag',
+                                     'dspace'])
+    parser.add_argument('--files', action='store_true',
+                        help='If set, start transfers from files as well as \
+                        folders.')
+    parser.add_argument('--hide', action='store_true',
+                        help='If set, hide the Transfers and SIPs in the \
+                        dashboard once they complete.')
+    parser.add_argument('-c', '--config-file', metavar='FILE',
+                        help='Configuration file(log/db/PID files)',
+                        default=None)
 
     # Logging
-    parser.add_argument('--verbose', '-v', action='count', default=0, help='Increase the debugging output.')
-    parser.add_argument('--quiet', '-q', action='count', default=0, help='Decrease the debugging output')
-    parser.add_argument('--log-level', choices=['ERROR', 'WARNING', 'INFO', 'DEBUG'], default=None, help='Set the debugging output level. This will override -q and -v')
+    parser.add_argument('--verbose', '-v', action='count',
+                        default=0, help='Increase the debugging output.')
+    parser.add_argument('--quiet', '-q', action='count',
+                        default=0, help='Decrease the debugging output')
+    parser.add_argument(
+        '--log-level', choices=['ERROR', 'WARNING', 'INFO', 'DEBUG'],
+        default=None, help='Set the debugging output level. This will \
+        override -q and -v')
 
     args = parser.parse_args()
 
-    log_levels = {
-        2: 'ERROR',
-        1: 'WARNING',
-        0: 'INFO',
-        -1: 'DEBUG',
-    }
-    if args.log_level is None:
-        level = args.quiet - args.verbose
-        level = max(level, -1)  # No smaller than -1
-        level = min(level, 2)  # No larger than 2
-        log_level = log_levels[level]
-    else:
-        log_level = args.log_level
+    log_level = loggingconfig.set_log_level(
+        args.log_level, args.quiet, args.verbose)
+
+    global CONFIG_FILE
```

Comment:
I think Python's `global` keyword should be avoided in general. I would like to avoid adding more `global`s to the codebase. Why not just add a `config_file` param to `get_setting`?

Reply:
Should be good now.